hera_mc/cm_dossier.py

```diff
@@ -0,0 +1,289 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2019 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""
+This contains the "Dossier" classes for Hookup, Parts and Connections
```

Comment:
It actually doesn't contain a class called "Dossier" at all and there's really only one class, not three as this docstring suggests.

Reply:
Right - the _intent_ is to migrate all three, but for now not.  I've changed the wording.